### DISCUSSION TITLE
feature request: cycle through screen layouts

### DISCUSSION THREAD

**Discussion by tristancg1995** [rafaelvcaetano/melonDS-android#1368] [Ideas]
I would like to be able to bind a button to cycle through my saved screen layouts
So I can alternate between what screen is larger without changing what side of the screen they are on

**Comment by JLiszt** [comment]
Greetings, 

I just wanted to revisit this request because this is something that could improve the emulator to a significant degree.

Azahar has this option and a lot of games benefit greatly from the ability to quickly change from a single screen view to a hybrid or side-by-side view. 

It's basically an improved, more flexible version of swapping between screens. Ideally, this could also be set up on a per-game basis, as some games benefit from certain layouts and other games benefit from different layouts.

A big thanks for everything, and especially for this awesome emulator!!!

PS Here's what the option looks like in Azahar.

<img width="1920" height="1080" alt="Azahar" src="https://github.com/user-attachments/assets/5c3290c4-7235-44d4-9817-81ca558460f7" />

**Comment by saidattax** [comment]
Ended up (vibe) coding a solution https://github.com/saidattax/melonDS-android/releases/tag/2.0.1-qol
